boards/arm/qemu_cortex_m0/qemu_cortex_m0.dts

```diff
@@ -10,7 +10,7 @@
 
 / {
 	model = "QEMU Cortex-M0";
-	compatible = "nordic,nrf51822-qfaa", "nordic,nrf51822";
+	compatible = "bbc,qemu-microbit";
```

Comment:
@ioannisg is this OK?

Reply:
I think so , yes

Reply:
qemu_cortex_m0 is emulating a bbc-micro bit

Reply:
that's why I called in qemu-microbit, otherwise I'd argue we could have just used the microbit board port.

Reply:
Yes. FWIW, the reason we did not use the microbit board port directly, is that QEMU does not emulate RTC but just the TIMER nRF peripheral, so we needed a new timer driver; this made me thing we could just have a separate qemu board with a local timer driver :)